dist/index.html

```diff
@@ -31,30 +34,58 @@
   </head>
 
   <body>
+    <!-- OutSystems change: branding -->
+    <div class="swagger-section">
+      <div id='header'>
+        <div class="swagger-ui-wrap">
+          <div class="os-header-right">
+            <div class="os-header-title">REST API Documentation</div>
+          </div>
+          <div class="os-header-left"><img class="os-logo-vertical" src="./imagesOutSystems/outsystems.png" alt="Outsystems"></div>
+          <!-- OutSystems change: customize header information
+			    <a id="logo" href="http://swagger.io">swagger</a> -->
+        </div>
+      </div>
+    </div>
     <div id="swagger-ui"></div>
 
-    <script src="./swagger-ui-bundle.js" charset="UTF-8"> </script>
-    <script src="./swagger-ui-standalone-preset.js" charset="UTF-8"> </script>
+    <script src="./swagger-ui-bundle.js" charset="UTF-8"></script>
+    <script src="./swagger-ui-standalone-preset.js" charset="UTF-8"></script>
     <script>
-    window.onload = function() {
-      // Begin Swagger UI call region
-      const ui = SwaggerUIBundle({
-        url: "https://petstore.swagger.io/v2/swagger.json",
-        dom_id: '#swagger-ui',
-        deepLinking: true,
-        presets: [
-          SwaggerUIBundle.presets.apis,
-          SwaggerUIStandalonePreset
-        ],
-        plugins: [
-          SwaggerUIBundle.plugins.DownloadUrl
-        ],
-        layout: "StandaloneLayout"
-      });
-      // End Swagger UI call region
+		  window.onload = function () {
+			  // Begin Swagger UI call region
+			  const ui = SwaggerUIBundle({
+			    // OutSystems change: change the URL!
+			    //It is supposed to change the URL!!!!!!!!!!!!!!!!!!!! This is for testing
+			    url: "https://os-jbfr253.domain.outsystems.com/swaggerUiDocumentation/rest/useCases/swagger.json",
```

Comment:
Forgot this?

Reply:
No no. Since we didn't join yet with the platform, it is hardcoded with a swagger that I have created (with my domain). We will integrate with the Platform, After having validations from UX team.